data/crac/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/remedial_action/network_action/PstSetpointTest.java

```diff
@@ -44,26 +47,52 @@ public void basicMethods() {
     }
 
     @Test
-    public void apply() {
+    public void applyStartsAtOne() {
+        pstSetpoint.rangeDefinition = RangeDefinition.STARTS_AT_ONE;
         Network network = NetworkImportsUtil.import12NodesNetwork();
         assertEquals(0, network.getTwoWindingsTransformer(networkElementId).getPhaseTapChanger().getTapPosition());
         pstSetpoint.apply(network);
         assertEquals(-5, network.getTwoWindingsTransformer(networkElementId).getPhaseTapChanger().getTapPosition());
     }
 
     @Test
-    public void applyOutOfBound() {
+    public void applyOutOfBoundStartsAtOne() {
         Network network = NetworkImportsUtil.import12NodesNetwork();
         PstSetpoint pstSetpoint = new PstSetpoint(
-            "out_of_bound",
-            new NetworkElement(networkElementId),
-            50);
+                "out_of_bound",
+                new NetworkElement(networkElementId),
+                50,
+                RangeDefinition.STARTS_AT_ONE);
+        try {
+            pstSetpoint.apply(network);
+            fail();
+        } catch (FaraoException e) {
+            assertEquals(String.format("Tap value 50 not in the range of high and low tap positions [-16,16] of the phase tap changer %s steps", networkElementId), e.getMessage());
```

Comment:
he message can be disturbing here, if you set your setpoint with STARTS_AT_ONE, you expect you PST to be [1, 33] for example. So I think you can adapt the message whether it starts at one or not.